lesson-10/jquery.js

```diff
@@ -0,0 +1,27 @@
+const express =require("express");
+const fs =require("fs");
+const gm =require("gm");
+const bodyparser =require("body-parser");
+const app =express();
+app.listen(3610);
+app.use(express.static("public"));
+app.use(bodyparser.json({limit: '2000mb'}));
+
+  app.post('/sendmessage',(req,res)=>{
+    fs.appendFile('/home/madhub/garagescript/lesson-10/public/notes.txt', `Name: ${req.body.memename} Comment: ${req.body.memecomment}`);
+  });
+
+  app.post('/pics', (req,res)=>{
```

Comment:
for the benefit of reading, change /pics to something like /writePicToMemeFiles, so does the others, so that when I see it it's much easier to guess and track. So when others help you debug it's even faster